news/1150.feature

```diff
@@ -0,0 +1 @@
+Yaml parser will now use `yaml.CSafeLoader` instead of `yaml.SafeLoader` whenever possible to speed up parsing.
```

Comment:
Minor reformatting
```suggestion
The YAML parser will now use `yaml.CSafeLoader` instead of `yaml.SafeLoader` whenever possible to speed up parsing
```